ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/S02_ManageContactItems.cs

```diff
@@ -431,6 +433,19 @@ public void MSOXWSCORE_S02_TC16_CreateContactItemWithInvalidItemClass()
                 2023,
                 @"[In t:ItemType Complex Type] If invalid values are set for these items in the CreateItem request, an ErrorObjectTypeChanged ([MS-OXWSCDATA] section 2.2.5.24) response code will be returned in the CreateItem response.");
         }
+
+        /// <summary>
+        /// This case is intended to validate the response returned by GetItem operation with the ItemShape element in which IncludeMimeContent element exists.
+        /// </summary>
+        [TestCategory("MSOXWSCORE"), TestMethod()]
+        public void MSOXWSCORE_S02_TC17_VerifyGetItemWithItemResponseShapeType_IncludeMimeContentBoolean()
+        {
+            Site.Assume.IsTrue(Common.IsRequirementEnabled(23091, this.Site), "[In Appendix C: Product Behavior] This element [MimeContent] is applicable for ContactType, TaskType and DistributionListType item when retrieving MIME content.(E2010SP3 and above follow this behavior.)");

```

Comment:
refine the description to "The MimeContent element is not applicable for ContactType, TaskType and DistributionListType item when retrieving MIME content."